src/screens/WebsocketResearchScreen/SignedChatScreen.tsx

```diff
@@ -3,41 +3,42 @@
 /* eslint-disable import/no-unresolved */
 
 import * as React from 'react';
+import ToastMessage from 'react-native-toast-message';
 import {FlatList, View} from 'react-native';
 
-import ToastMessage from 'react-native-toast-message';
 import BaseChatItem from '../../components/AnonymousChat/BaseChatItem';
 import ChatDetailHeader from '../../components/AnonymousChat/ChatDetailHeader';
+import ChatReplyPreview from '../../components/AnonymousChat/child/ChatReplyPreview';
 import InputMessageV2 from '../../components/Chat/InputMessageV2';
-import {Context} from '../../context';
-import {setChannel} from '../../context/actions/setChannel';
+import Loading from '../Loading';
+import useChatScreenHook from '../../hooks/screen/useChatScreenHook';
+import useMessageHook from '../../hooks/screen/useMessageHook';
 import useMoveChatTypeHook from '../../hooks/core/chat/useMoveChatTypeHook';
-import {SIGNED} from '../../hooks/core/constant';
 import useProfileHook from '../../hooks/core/profile/useProfileHook';
-import useChatScreenHook from '../../hooks/screen/useChatScreenHook';
+import {Context} from '../../context';
+import {SIGNED} from '../../hooks/core/constant';
 import {getChatName} from '../../utils/string/StringUtils';
-import Loading from '../Loading';
+import {setChannel} from '../../context/actions/setChannel';
 import {styles} from './SampleChatScreen';
 
 const SignedChatScreen = () => {
+  const flatlistRef = React.useRef<FlatList>();
   const {
     selectedChannel,
     chats,
     goBackFromChatScreen,
     goToChatInfoScreen,
     sendChat,
-    updateChatContinuity,
-    loadingChat
+    updateChatContinuity
   } = useChatScreenHook(SIGNED);
+  const {replyPreview, clearReplyPreview} = useMessageHook();
+  const {moveToAnonymousChannel} = useMoveChatTypeHook();
+  const {signedProfileId} = useProfileHook();
+  const updatedChats = updateChatContinuity(chats);
 
-  const flatlistRef = React.useRef<FlatList>();
   const [loading, setLoading] = React.useState(false);
   const [, dispatchChannel] = (React.useContext(Context) as unknown as any).channel;
   const [profile] = (React.useContext(Context) as unknown as any).profile;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Casting the context to `any` can lead to potential type errors in the future as it bypasses TypeScript's type checking. It would be better to define an interface that describes the shape of your context and use that instead.



<!-- This is an auto-generated comment by OSS CodeRabbit -->